mappings/net/minecraft/client/gui/screen/realms/RealmsCreateRealmScreen.mapping

```diff
@@ -1,17 +1,29 @@
 CLASS net/minecraft/unmapped/C_pmrmyvxc net/minecraft/client/gui/screen/realms/RealmsCreateRealmScreen
+	FIELD f_guauegxt CONTENT_WIDTH I
 	FIELD f_hmrzumvk nameBox Lnet/minecraft/unmapped/C_vopzcnhf;
 	FIELD f_jgoisftz callback Ljava/lang/Runnable;
 	FIELD f_qbffzfvz descriptionBox Lnet/minecraft/unmapped/C_vopzcnhf;
-	FIELD f_srokocfw WORLD_NAME Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_srokocfw WORLD_NAME_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
```

Comment:
```suggestion
	FIELD f_srokocfw WORLD_NAME Lnet/minecraft/unmapped/C_rdaqiwdt;
```
the text suffixes were dropped in #464
(or would this conflict with something?)

Reply:
Right, I completely forgot about that...

IMO, looking at a lot of the text related names, the text suffix makes the purpose of the field a lot clearer. For instance, `WORLD_NAME` isn't the same as `WORLD_NAME_TEXT`, the former could be interpreted as a string for a hardcoded world name, while the latter tells me it's a text that says "World name" or something similar.

I'll remove the suffix(es), since the discussion already happened elsewhere and we can discuss it later anyway